src/common/Pagination/index.js

```diff
@@ -92,9 +114,21 @@ export const Pagination = (isMoviesPage) => {
                 <CounterNumber>{totalPages}</CounterNumber>
             </Counter>
             <ButtonsContainer>
-                {renderButton(() => changePage(currentPage + 1), currentPage === totalPages, null, <PointerRight />, "Next")}
-                {renderButton(() => changePage(totalPages), currentPage === totalPages, <PointerRight />, <PointerRight />, "Last")}
+                {renderButton(
```

Comment:
It would be nice when you are at the first page to add to button "previous" or "first" disabled attribute. Same with "last".